### PR TITLE
Rework project organization

### PR DESCRIPTION
**Pull request by jffng**
This PR attempts to make the theme easier to understand and maintain by introducing some minimal boilerplate and optimizing a few areas. The functionality / behavior of the theme should not have changed at all.

In particular:
- The front-end styles are broken out into base and block styles
- The CSS variables are consolidated into a single location
- `add_editor_style` is used to enqueue styles meant specifically for the editor, so they are not loaded unnecessarily and eliminating the need to target `.editor-styles-wrapper`
- The theme fonts requests are consolidated

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -65,54 +74,36 @@ function setup() : void {
...
+	$font_families = array();
+	$font_families[] = 'family=Poppins:wght@700;900';
+	$font_families[] = 'family=Francois+One';
+	$font_families[] = 'family=Karla:ital,wght@0,400;0,700;1,400;1,700';
+	$font_families[] = 'display=swap';
+
+	// Make a single request for the theme fonts
+	$fonts_url = 'https://fonts.googleapis.com/css2?' . implode( '&', $font_families );
+
+	return $fonts_url;
```
Comment (ChaosExAnima) [inline functions.php:108]: One reason I enqueued the font families multiple times was I found the additional `family=` query var was being removed by WP's URL sanitization, so only one was left. Have you seen that?

Comment (jffng) [reply functions.php:108]: I did encounter that! Couldn't figure out how to get this to work with `add_query_arg()`, so I went with a different approach.

Comment (ChaosExAnima) [reply functions.php:108]: Oh interesting, this works? Interesting. Happy to merge if it looks good on your end 👍

Comment (ChaosExAnima) [reply functions.php:108]: Yep, just tested and it's okay. 🚢

**Pull request merged**